sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/rules/QueryExecutionMetering.scala

```diff
@@ -64,10 +64,13 @@ case class QueryExecutionMetering() {
   /** Dump statistics about time spent running specific rules. */
   def dumpTimeSpent(): String = {
     val map = timeMap.asMap().asScala
-    val maxLengthRuleNames = if (map.isEmpty) {
-      0
-    } else {
-      map.keys.map(_.toString.length).max
+    val maxLengthRuleNames = {
```

Comment:
Did this fix anything? Looks like equivalent logic and would be ever so slightly slower

Reply:
Oops, looks like it was already fixed in master.